etc/kayobe/ansible/gendash.yml

```diff
@@ -0,0 +1,68 @@
+- hosts: localhost
+  vars: 
+    ansible_python_interpreter: /usr/bin/python3
+  tasks:
+    - name: Include the generated kolla.yml
+      include_vars:
+        file: ../../kolla/globals.yml
```

Comment:
When you run this as a Kayobe custom playbook, you should have access to service flags without having to include this. Eg. `kolla_enable_foo` etc.